test/Microsoft.Windows.CsWin32.Tests/GeneratorTests.cs

```diff
@@ -446,6 +448,8 @@ internal partial struct MainAVIHeader
         internal struct __dwReserved_4
         {
             internal uint _0, _1, _2, _3;
+            /// <summary>Always <c>4</c>.</summary>
+            internal int Length => 4;
```

Comment:
Lets make it a constant rather than a property. That may unblock compiler optimizations.

Reply:
How would you use a constant, syntactically?

Reply:
It wouldn't change the syntax for usage at all. I'm merely suggesting we change this:

```diff
-internal int Length => 4;
+internal const int Length = 4;
```

So callers are unaffected, but it can now be used in more places (e.g. attributes) and compilers or the JIT may optimize based on knowing the value never changes.

Reply:
I understand that part about constants, but don't constants and static members preclude the syntax `header.dwReserved.Length` by requiring qualification by the class name instead of instance invocation?

Reply:
On phone, but I checked sharplab just to be sure I wasn't totally blanking somehow and it gave me "error CS0176: Member cannot be accessed with an instance reference; qualify it with a type name instead."

Reply:
Oh, I see what you're getting at. I'd forgotten about that impact. Ok, we can keep what you've got.